dev_guide/cron_jobs.adoc

```diff
@@ -45,8 +45,10 @@ endif::[]
 [WARNING]
 ====
 A cron job creates a job object approximately once per execution time of its
-schedule, but there are circumstances in which it will fail to create a job or
-two jobs might be created. As a result, jobs should be idempotent.
+schedule, but there are circumstances in which it fails to create a job or
+two jobs might be created.  Therefore, jobs should be idempotent and you
+should monitor and xref:#cleaning-up-after-a-cron-job[clean cron jobs]
```

Comment:
`xref:#cleaning-up-after-a-cron-job[clean cron jobs]` should be `xref:cleaning-up-after-a-cron-job[clean cron jobs]` please remove the `#` sign.